tendenci/apps/events/models.py

```diff
@@ -881,16 +881,27 @@ def can_edit_child_events(self):
         if not self.event.nested_events_enabled:
             return False
 
+        can_edit = False
         for registrant in self.registrant_set.filter(cancel_dt__isnull=True):
-            return (
+            can_edit = (
                 not registrant.registration_closed and (
                     (registrant.user and registrant.user.profile.is_superuser and
                     registrant.event.child_events.exists()) or
                     registrant.event.upcoming_child_events.exists()
                 )
             )
+            if can_edit:
+                return True
 
         return False
+    
+    @property
+    def can_admin_edit_child_events(self):
+        """If admin user can edit child events, return True"""
+        if not self.event.nested_events_enabled:
+            return False
+
+        return self.event.child_events.exists()
```

Comment:
This turns on the button to Save and Edit Sessions for admins (if nested events are enabled & event has child events)